conda-store-server/conda_store_server/app.py

```diff
@@ -72,6 +73,9 @@ def conda_store_validate_action(
         )
 
 
+CONDA_STORE_DIR = Path.home() / ".conda-store"
```

Comment:
This needs to be moved to storage (to avoid import errors) and used there in `storage_path` as well.